src/Sulu/Component/Rest/ListBuilder/Metadata/Resources/schema/list-2.0.xsd

```diff
@@ -72,18 +72,20 @@
         <xs:attributeGroup ref="fieldAttributeGroup"/>
     </xs:complexType>
 
-    <xs:complexType name="parametersType">
+    <xs:complexType name="filterType">
         <xs:sequence>
-            <xs:element type="parameterType" name="parameter" minOccurs="1" maxOccurs="unbounded"/>
+            <xs:element type="paramType" name="param" minOccurs="0" maxOccurs="unbounded"/>
         </xs:sequence>
+        <xs:attributeGroup ref="filterAttributeGroup"/>
     </xs:complexType>
 
-    <xs:complexType name="parameterType">
-        <xs:simpleContent>
-            <xs:extension base="xs:string">
-                <xs:attribute name="key" type="xs:string"/>
-            </xs:extension>
-        </xs:simpleContent>
+    <xs:complexType name="paramType">
+        <xs:sequence>
+            <xs:element type="paramType" name="param" minOccurs="0" maxOccurs="unbounded"/>
+        </xs:sequence>
+        <xs:attribute name="name" type="xs:string"/>
```

Comment:
in the `properties-1.0.xsd` we declared this as required

Reply:
And it has already caused problems. Sometimes just value would make more sense, especially in the context of a collection. Therefore I would not start with the same mistake again.

Reply:
did not know about that 🙂 in which case does it make sense to have just a value?

Reply:
e.g. something like this:

```xml
<param name="dropdown-options" type="collection">
    <param value="1" />
    <param value="2" />
    <!-- ... -->
</param>
```

Reply:
okay i think we need to adjust the logic of the `ListXmlLoader` a bit then?